api/leaderworkerset/v1/leaderworkerset_types.go

```diff
@@ -39,11 +39,11 @@ const (
 	// statefulset/pod belong to.
 	GroupIndexLabelKey string = "leaderworkerset.sigs.k8s.io/group-index"
 
-	// Worker index will be added to pods as a label and annotation which is
+	// Worker index will be added to pods as a label which is
 	// the index/identity of the pod in the group.
 	WorkerIndexLabelKey string = "leaderworkerset.sigs.k8s.io/worker-index"
 
-	// Size will be added to leader pods as an annotation which corresponds to
```

Comment:
I guess this will result in other changes once we run `make generate`? I guess.

Reply:
Thanks
There are no files that need to be changed after the `make generate`